templates/l_to_l_conversion.cpp.in

```diff
@@ -17,40 +17,40 @@
 // ====================================================================
 
 /**
- * @file @ModelName@_f_to_f_conversion.cpp
+ * @file @ModelName@_l_to_l_conversion.cpp
  *
- * This file was generated with FlexibleSUSY @FlexibleSUSYVersion@ and SARAH @SARAHVersion@ .
+ * This file was generated at @DateAndTime@ with FlexibleSUSY
+ * @FlexibleSUSYVersion@ and SARAH @SARAHVersion@
  */
 
 #include <valarray>
 #include <complex>
+#include <iostream>
```

Comment:
@uukhas , sorry, I've just stumbled upon this include. Can you please check whether it can be removed? (It seems so.)

Reply:
Thank you very much, @Expander. I think that this iostream is chasing me everywhere.

Reply:
Thanks for removing it! :)
Yes, it would be nice if the compiler would tell us which header inclusions are superfluous...

Reply:
I searched and found quite a lot of software (at first look), which can test for not used `#include` files, such as https://github.com/myint/cppclean. Can they be used at, for example `travis`-phase?

Reply:
Cool, thanks, I didn't know of this tool. (I was curious what https://github.com/include-what-you-use/include-what-you-use could do, but it seems to require CMake.)

We'd have to play around with it a bit. But if it works for us, then I'd be very happy to create a github action for it (or similar).

Reply:
Have you tried it?

Reply:
I made `SM` folder and typed 
```
cppclean --include-path=src --include-path=model_specific/SM --include-path=config models/SM
```

for the check and got a lot of warnings such as:
```
models/SM/SM_high_scale_constraint.hpp:27: 'flexiblesusy::SM_high_scale_constraint' not used
```
as well as a lot of exceptions during the procession of files.

Reply:
~~~
models/SM/SM_high_scale_constraint.hpp:27: 'flexiblesusy::SM_high_scale_constraint' not used
~~~
Interesting. Although, I think this is a false-positive, because we're using a template specialization of `SM_high_scale_constraint`, so we are in fact using it.

Reply:
For curiosity, I've run it on `GM2Calc`, and got a few messages, of which all were incorrectly pointing to missing includes. It seems that cppcheck is confused for example by namespace prefixes and functions that involve templated types.

Reply:
I see.. So one just need to be really careful with this software. But at least we can now almost definitely say, that this one is not what we need.

Reply:
indeed. :)